src/components/input-field/partial-styles/trailing-icon.scss

```diff
@@ -16,7 +16,7 @@
         }
     }
     &:hover {
-        .trailing-icon {
+        .mdc-text-field__icon--trailing {
             box-shadow: var(--button-shadow-normal);
```

Comment:
These full-width readonly trailing icons do not get box-shadow on hover. The result is that case of type `email`, there is no visual effect to communicate that the entire field is clickable.


https://user-images.githubusercontent.com/35954987/125903965-72ef0c8f-f335-4882-b04d-4b0746517bb9.mov

(generally, the way they get box-shadow must be improved, for example when invalid, they may show a red shadow or border or similar).

Reply:
The email-button could get a light grey colour perhaps?

If you push what you have, I can have a look.

Reply:
⚡

Reply:
in the old version, the links get a hover effect, styles are already there. But they won't show up here.

Reply:
Did you try my latest changes?

Reply:
Ah awesome! I read your comment in the code as well 🤩 Great job!

This page wasn't refreshed automatically. So I missed that there are new changes, until I wanted to push mine on slider, and then I saw I must pull ;)